platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Transform.java

```diff
@@ -60,21 +55,15 @@ void initialise(@NonNull MapboxMap mapboxMap, @NonNull MapboxMapOptions options)
 
   @UiThread
   public final CameraPosition getCameraPosition() {
-    if (cameraPosition == null) {
-      cameraPosition = invalidateCameraPosition();
+    if(cameraPosition == null){
+      throw new IllegalStateException();
```

Comment:
Not sure if this should be the way to go. Since the initial place where we fetch the camera position from the core is on frame fully rendered, the `cameraPosition` is still `null` when we deliver the `onMapReady`, as the frame is typically not fully render yet. I can reproduce this by calling `mapboxMap#getCameraPosition` inside of the `onMapReady`.

We should either keep this guard, or invalidate the camera before delivering the `onMapReady`.

Reply:
Capturing from the chat that this is only a test setup, to let the CI catch possible regressions.